cocos/spine/lib/spine-core.d.ts

```diff
@@ -421,8 +421,8 @@ declare namespace spine {
         start = 0,
         interrupt = 1,
         end = 2,
-        dispose = 3,
-        complete = 4,
+        complete = 3,
+        dispose = 4,
```

Comment:
The enum value changes break the compatibility. Need to describe it in Release Note.